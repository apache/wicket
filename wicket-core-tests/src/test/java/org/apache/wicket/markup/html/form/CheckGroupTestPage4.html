<html>
<head><title>CheckGroupTestPage4</title></head>
<body>
	<form wicket:id="form">
		<span wicket:id="group">
			<input type="checkbox" wicket:id="check1">check1</input>
			<span wicket:id="container">
				<input type="checkbox" wicket:id="check2">check2</input>
			</span>
		</span>
	</form>
</body>
</html>
