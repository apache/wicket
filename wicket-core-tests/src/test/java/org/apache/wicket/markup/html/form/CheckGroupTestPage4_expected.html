<html>
<head><title>CheckGroupTestPage4</title></head>
<body>
	<form wicket:id="form" id="forma" method="post" action="./org.apache.wicket.markup.html.form.CheckGroupTestPage4?3-1.-form"><div id="forma_hf_0" hidden="" class="hidden-fields"></div>
		
			<input type="checkbox" wicket:id="check1" id="check1b" name="group" value="check0" checked="checked">check1</input>
			<span wicket:id="container">
				<input type="checkbox" wicket:id="check2" id="check2c" name="group" value="check1" checked="checked">check2</input>
			</span>
		
	</form>
</body>
</html>
